Core/clim-basic/geometry/regions.lisp

```diff
@@ -53,13 +53,26 @@
 
 (in-package :clim-internals)
 
-(defclass nowhere-region (region nowhere-mixin) ())
+;;;; Design <-> Region Equivalences
+
+;;; As Gilbert points in his notes, transparent ink is in every
+;;; respect interchangable with the nowhere region, and likewise
+;;; foreground ink is interchangable with the everywhere region.
+;;; By defining the following mixins and adding them to the
+;;; appropriate ink/region class pairs, we can reduce the number
+;;; of methods necessary.
```

Comment:
"of methods necessary in design module".

Reply:
Fixed.